### PR TITLE
Do not localize Language Server names

### PR DESCRIPTION
**Pull request by NTaylorMullen**
- I imagine you wouldn't want to localize all of the other language clients but didn't want to make the changes there under an assumption.

Part of dotnet/aspnetcore#29619

### CODE REVIEW

File: src/VisualStudio/Core/Def/Implementation/LanguageClient/RazorInProcLanguageClient.cs [hunk 1]
```diff
@@ -38,7 +38,7 @@ internal class RazorInProcLanguageClient : AbstractInProcLanguageClient
         /// <summary>
         /// Gets the name of the language client (displayed to the user).
```
Comment (NTaylorMullen) [inline src/VisualStudio/Core/Def/Implementation/LanguageClient/RazorInProcLanguageClient.cs:39]: Should I remove the "displayed to the user" portion? Given that it's only in yellow bars 😆

Comment (allisonchou) [reply src/VisualStudio/Core/Def/Implementation/LanguageClient/RazorInProcLanguageClient.cs:39]: Maybe we can specify that it's used just for yellow bar cases?

**Review by allisonchou — APPROVED** [review]
🎉

File: src/VisualStudio/Core/Def/ServicesVSResources.resx [hunk 2]
```diff
@@ -1441,9 +1441,6 @@ I agree to all of the foregoing:</value>
   <data name="Prefer_simplified_boolean_expressions" xml:space="preserve">
     <value>Prefer simplified boolean expressions</value>
   </data>
-  <data name="Razor_CSharp_Language_Server_Client" xml:space="preserve">
```
Comment (dibarbet) [inline src/VisualStudio/Core/Def/ServicesVSResources.resx:1444]: I think the all the translations should be removed as well

**Review by dibarbet — COMMENTED** [review]
>     * I imagine you wouldn't want to localize all of the other language clients but didn't want to make the changes there under an assumption.

I think if we're going to do it for one, we should do it for all.

File: src/VisualStudio/Core/Def/ServicesVSResources.resx (hunk 2 above)
Comment (NTaylorMullen) [reply src/VisualStudio/Core/Def/ServicesVSResources.resx:1444]: Where do you all store the translations?

**Comment by NTaylorMullen** [comment]
> I think if we're going to do it for one, we should do it for all.

Sure I can update. Inclusive of XAMLs?

**Comment by dibarbet** [comment]
> 
> 
> > I think if we're going to do it for one, we should do it for all.
> 
> Sure I can update. Inclusive of XAMLs?

I think that's fine, but @mgoertz-msft to confirm

File: src/VisualStudio/Core/Def/ServicesVSResources.resx (hunk 2 above)
Comment (dibarbet) [reply src/VisualStudio/Core/Def/ServicesVSResources.resx:1444]: Here - https://github.com/dotnet/roslyn/tree/master/src/VisualStudio/Core/Def/xlf - all the xlf with the same name (ServicesVSResources)

I think a commandline build should fix them?  If not there was some msbuild flag to pass to regenerate

Comment (dibarbet) [reply src/VisualStudio/Core/Def/ServicesVSResources.resx:1444]: I think /t:UpdateXlf

**Comment by NTaylorMullen** [comment]
🆙 📅 

> I think that's fine, but @mgoertz-msft to confirm

I went ahead and did the needful and included in this PR but wont merge until I get confirmation.

**Review by dibarbet — APPROVED** [review]

**Comment by mgoertz-msft** [comment]
Thanks. I'm all for consistency, so XAML should do what everyone else does.

**Review by mgoertz-msft — APPROVED** [review]

**Review by allisonchou — APPROVED** [review]

**Review by davidwengier — APPROVED** [review]

**Pull request merged**